x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_setup_trust/remote_cluster_setup_trust.tsx

```diff
@@ -0,0 +1,182 @@
+    {
+      defaultMessage:
+        'This is our easier to setup method, but less secure than the API key based approach.',
+    }
+  ),
+};
+
+const docLinks = {
+  apiKey: remoteClustersUrl,
+  apiKeyCloud: remoteClustersUrl,
+  cert: remoteClustersUrl,
+  certCloud: remoteClustersUrl,
+};
+
+interface Props {
+  onBack: () => void;
+  onSubmit: () => void;
+  isSaving: boolean;
+}
+
+export const RemoteClusterSetupTrust = ({ onBack, onSubmit, isSaving }: Props) => {
+  const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
+  const { canUseAPIKeyTrustModel, isCloudEnabled } = useContext(AppContext);
+
+  return (
+    <div>
+      <EuiText size="m" textAlign="center">
+        <p>
+          <FormattedMessage
+            id="xpack.remoteClusters.clusterWizard.trustStep.title"
+            defaultMessage="To establish trust between this deployment and your remote cluster, {br} choose one of the following options."
+            values={{
+              br: <br />,
+            }}
+          />
+        </p>
+      </EuiText>
+
+      <EuiSpacer size="xxl" />
+
+      <EuiFlexGroup wrap justifyContent="center">
+        {canUseAPIKeyTrustModel && (
+          <EuiFlexItem style={{ maxWidth: CARD_MAX_WIDTH }}>
+            <EuiCard
+              title={i18nTexts.apiKeyTitle}
+              paddingSize="l"
+              betaBadgeProps={{ label: i18nTexts.apiKeyBadge, color: 'accent' }}
+              data-test-subj="setupTrustApiKeyCard"
+            >
+              <EuiText size="s">
+                <p>{i18nTexts.apiKeyDescription}</p>
+              </EuiText>
+              <EuiSpacer size="xl" />
+              <EuiButtonEmpty
+                href={isCloudEnabled ? docLinks.apiKeyCloud : docLinks.apiKey}
+                iconSide="right"
+                iconType="popout"
+                target="_blank"
+                data-test-subj="setupTrustApiKeyCardDocs"
+              >
+                <FormattedMessage
+                  id="xpack.remoteClusters.clusterWizard.trustStep.docs"
+                  defaultMessage="Read more"
+                />
+              </EuiButtonEmpty>
+            </EuiCard>
+          </EuiFlexItem>
+        )}
+
+        <EuiFlexItem style={{ maxWidth: CARD_MAX_WIDTH }}>
+          <EuiCard title={i18nTexts.certTitle} paddingSize="l" data-test-subj="setupTrustCertCard">
+            <EuiText size="s">
+              <p>{i18nTexts.certDescription}</p>
+            </EuiText>
+            <EuiSpacer size="xl" />
+            <EuiButtonEmpty
+              href={isCloudEnabled ? docLinks.certCloud : docLinks.cert}
+              iconSide="right"
+              iconType="popout"
+              target="_blank"
+              data-test-subj="setupTrustCertCardDocs"
+            >
+              <FormattedMessage
+                id="xpack.remoteClusters.clusterWizard.trustStep.docs"
+                defaultMessage="Read more"
```

Comment:
It is not mandatory. It's more like a suggested approach. If the user doesn't follow it, the cluster will still be created.

@florent-leborgne if you feel this is a more actionable CTA then by all means, let's use the proposed text.

Reply:
But it seems that we really want them to follow the docs and set it up at that moment, since we after have the modal where we're asking them to confirm they did set it up, so I think it makes more sense to have the doc links as an actionable CTA.

Reply:
I've made the copy changes and transformed the links into ctas

![Screenshot 2023-07-26 at 07 31 24](https://github.com/elastic/kibana/assets/1191206/ab9a95a1-8de2-424a-94ab-dbe2fc250c64)